src/OrchardCore/OrchardCore.Sms.Core/Services/TwilioSmsProvider.cs

```diff
@@ -14,6 +14,10 @@ namespace OrchardCore.Sms.Services;
 
 public class TwilioSmsProvider : ISmsProvider
 {
+    public const string Name = "Twilio";
```

Comment:
Maybe useful to also define the `Name` property in the `ISmsProvider` interface.

Reply:
Added a localized name.